nextcord/asset.py

```diff
@@ -220,6 +220,19 @@ def _from_avatar(cls, state, user_id: int, avatar: str) -> Asset:
             animated=animated,
         )
 
+    @classmethod
+    def _from_avatar_decoration(cls, state, user_id: int, decoration: str) -> Asset:
+        animated = decoration.startswith("a_")
+        # some decorations are animated but they are returned as an animated png, -
+        # - you can't get them as a gif (like stickers)
+        # their hashes start with a_
+        return cls(
+            state,
+            url=f"{cls.BASE}/avatar-decorations/{user_id}/{decoration}.png?size=1024",
+            key=decoration,
+            animated=animated,
```

Comment:
`animated` being set to `True` informs `Asset.replace` that getting a gif version of an avatar decoration is allowed, when it is not.